src/components/Exchange/SwapBox.js

```diff
@@ -783,8 +783,15 @@ export default function SwapBox(props) {
     if (!fromTokenInfo || !fromTokenInfo.minPrice) {
       return [t`Incorrect network`];
     }
-    if (fromTokenInfo && fromTokenInfo.balance && fromAmount && fromAmount.gt(fromTokenInfo.balance)) {
-      return [t`Insufficient ${fromTokenInfo.symbol} balance`];
+
+    if (
+      !savedShouldDisableOrderValidation &&
```

Comment:
can we use either separate setting or rename this one? not sure which option is better, we can discuss how we can handle such things in future